cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
 ** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
 Container Platform 4 on RHEL CoreOS 8_.
 
+=== Web console
+// CNV-4076 UI view/config dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
```

Comment:
Suggest revision to make it active:
`You can now view and configure dedicated resources for virtual machines in the *Virtual Machine Details* page.`
It's a web console improvement so we don't need to repeat it here.

Reply:
Done,